### PR TITLE
Update ior's page

### PR DESCRIPTION
**Pull request by niklasvatn**

### CODE REVIEW

File: en/clubs/informationsorganet/body.md [hunk 1]
```diff
@@ -1 +1,23 @@
-[I want access to Datasektionen's GitHub](https://dsekt.se/github-access)
\ No newline at end of file
+# IOR
+
+We are happy that you found IOR's page. We are also known as the _Donkeys_, which is a name that does not describe what we do very well. IOR is in charge of developing new systems and maintaining the Chapter’s current systems. In short, we are a group of students who enjoy programming and learning together. Everyone is welcome here, regardless of whether you have worked as a developer for a few years or are completely new to programming.
+
```
Comment (elmaxe) [inline en/clubs/informationsorganet/body.md:4]:
```suggestion
<style>
img {
  width: 500px;
}
</style>

<image src="https://static.wikia.nocookie.net/disney/images/1/1c/Profile_-_Eeyore.png/revision/latest?cb=20210516060155" />
```

File: en/clubs/informationsorganet/body.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+## The Chapter’s systems
+
+The chapter’s systems are available on our [Github](https://dsekt.se/github-link-website), if you want access to see or edit something, you can request access through this [form] (https:// dsekt.se/ github-access).
```
Comment (elmaxe) [inline en/clubs/informationsorganet/body.md:7]: space in between https and dsekt

File: namnder/informationsorganet/body.md [hunk 3]
```diff
@@ -1 +1,23 @@
-[Jag vill ha access till Datasektionens GitHub](https://dsekt.se/github-access)
\ No newline at end of file
+# IOR
+
+Kul att du hittat till IOR:s sida. Vi är även kända som _Informationsorganet_, vilket är ett namn som inte beskriver vår verksamhet särskilt bra. Det IOR sysslar med är utveckling av nya system och underhållning av Datasektionens nuvarande system. Kort och gott, vi är ett gäng studenter som tycker om att programmera och lära tillsammans. Här är alla välkomna, oavsett om du jobbat som utvecklare i några år eller är helt ny till programmering.
+
```
Comment (elmaxe) [inline namnder/informationsorganet/body.md:4]:
```suggestion
<style>
img {
  width: 500px;
}
</style>

<image src="https://static.wikia.nocookie.net/disney/images/1/1c/Profile_-_Eeyore.png/revision/latest?cb=20210516060155" />
```

**Review by elmaxe — CHANGES_REQUESTED** [review]

File: en/clubs/informationsorganet/body.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+
+Here is a list of some systems that previous students have built and that the chapter use extensively:
+
+- [Cashflow](https://cashflow.datasektionen.se) is probably our most used system. Here you can register expenses that you have made for the chapter and the Treasurer (or some other responsible person) will pay the money to you as soon as possible. Neat, right? [Mentometer](https://mentometer.datasektionen.se) is used to hold certain elections during chapter meetings (SM).
+- [aaallt](https://aaallt.datasektionen.se/) may not be our most exciting system, but here you can go to almost every other system the chapter has.
+
+### Current projects (updated November 2021)
+
+- Budget HD: Our current budget system has its problems and therefore the development of a new, better (hopefully) system has begun, in HD.
+- dUrn 2.0: Our current system for ballot-elections (see [the regulations](https://styrdokument.datasektionen.se/reglemente) 3.12.7 for more information) needs to be reworked a bit. The system is being rebuilt in Go and is intended to be ready for the elections at Glögg-SM 2021.
```
Comment (brynolv) [inline en/clubs/informationsorganet/body.md:19]: Can we link directly to sections in the regulations? If not, that might be something to look into.

**Review by brynolv — DISMISSED** [review]
The space elmaxe mentioned, otherwise it looks gtm

File: en/clubs/informationsorganet/body.md (hunk 4 above)
Comment (elmaxe) [reply en/clubs/informationsorganet/body.md:19]: Yes: `https://styrdokument.datasektionen.se/reglemente#3-12-7-urnval`

Comment (elmaxe) [reply en/clubs/informationsorganet/body.md:19]: Bar is in the way though

**Review by elmaxe — APPROVED** [review]

**Pull request merged**